index.html

```diff
@@ -1,325 +1,312 @@
-                        css
-                     </p>
-                     <p class="works-card-topic">
-                        javascript
-                     </p>
-                  </div>
-                  <div class="works-card-btn">
-                     <button type="button" class="btn">
-                     See Project
-                     </button>
-                  </div>
-               </div>
+            <li class="about-lang-row">
+              <img src="./assets/css.svg.svg" alt="javascript icon" />
+              <p class="about-lang-name">CSS</p>
             </li>
-
-            <li class="works-card">
-               <img class="works-card-img" src="./assets/multi_post_1.svg" alt="tonic project">
-               <div class="works-card-text">
-                  <h2 class="works-card-title">
-                     Multi-Post Stories
-                  </h2>
-                  <div class="works-card-sub">
-                     <p class="sub-1">
-                        CANOPY
-                     </p>
-                     <ul class="works-card-sub-list">
-                        <li class="works-card-sub-link">
-                           Backend Dev
-                        </li>
-                        <li class="works-card-sub-link">
-                           2015
-                        </li>
-                     </ul>
-                  </div>
-                  <p class="works-card-desc">
-                     A daily selection of privately personalized reads; no accounts or sign-ups required.
-                  </p>
-                  <div class="works-card-topics-list">
-                     <p class="works-card-topic">
-                        html
-                     </p>
-                     <p class="works-card-topic">
-                        css
-                     </p>
-                     <p class="works-card-topic">
-                        javascript
-                     </p>
-                  </div>
-                  <div class="works-card-btn">
-                     <button type="button" class="btn">
-                     See Project
-                     </button>
-                  </div>
-               </div>
+          </ul>
+        </li>
+        <li class="about-tool border-bottom">
+          <div class="about-languages-list">
+            <h3 class="about-lang-title">Frameworks</h3>
+            <img
+              class="about-lang-img"
+              src="./assets/chev-right.png"
+              alt="arrow icon"
+            />
+          </div>
+        </li>
+        <li class="about-tool border-bottom">
+          <div class="about-languages-list">
+            <h3 class="about-lang-title">Skills</h3>
+            <img
+              class="about-lang-img"
+              src="./assets/chev-right.png"
+              alt="arrow icon"
+            />
+          </div>
+        </li>
+      </ul>
+    </section>
+    <section class="Contact-form">
```

Comment:
- [ ] Instead of `<section>` element here, Please use `<footer>` tag. It will make sure your code is semantic and follows HTML best practices.

Reply:
Thanks just made the required changes